go/tools/builders/env.go

```diff
@@ -133,18 +133,14 @@ func (e *env) runCommand(args []string) error {
 	cmd := exec.Command(args[0], args[1:]...)
 	// Redirecting stdout to stderr. This mirrors behavior in the go command:
 	// https://go.googlesource.com/go/+/refs/tags/go1.15.2/src/cmd/go/internal/work/exec.go#1958
-	cmd.Stdout = os.Stderr
-	cmd.Stderr = os.Stderr
-	return runAndLogCommand(cmd, e.verbose)
+	return runAndLogCommandOutput(cmd, os.Stderr, os.Stderr, e.verbose)
```

Comment:
Instead of introducing `runAndLogCommandOutput`, I think it would be simpler to add the relativization here, especially since `runCommandToFile` should not use use it. Something like:

```
buf := &bytes.Buffer{}
cmd := exec.Command(args[0], args[1:]...)
// Redirecting stdout to stderr. This mirrors behavior in the go command:
// https://go.googlesource.com/go/+/refs/tags/go1.15.2/src/cmd/go/internal/work/exec.go#1958			cmd.Stdout = os.Stderr	
cmd.Stdout = buf
cmd.Stderr = buf
err := runAndLogCommand(cmd, e.verbose)
fmt.Fprint(os.Stderr, relativizePaths(buf.Bytes())
return err
```

Reply:
LGTM! Updated